crates/nu-command/src/formats/from/toml.rs

```diff
@@ -23,6 +24,19 @@ impl Command for FromToml {
         "Parse text as .toml and create record."
     }
 
+    fn run(
+        &self,
+        __engine_state: &EngineState,
```

Comment:
Probably just need 1 underscore here.
```suggestion
        _engine_state: &EngineState,
```